internal/graph/check.go

```diff
@@ -659,11 +667,15 @@ func (c *LocalChecker) buildCheckAssociatedObjects(req *ResolveCheckRequest, obj
 			})
 		}
 
+		objectIDsArrayOfStrings := make([]string, objectIDs.Size())
```

Comment:
Why not just call 
```
objectIDsArrayOfStrings = objectIDs.Values()
```
?

Reply:
Hehe because it doesn't work:

```go
objectIDsArrayOfStrings := objectIDs.Values().([]string)
// Invalid type assertion: objectIDs.Values().([]string) (non-interface type []interface{} on the left)
```